mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -638,6 +638,35 @@ func resourceDataprocCluster() *schema.Resource {
 											},
 										},
 									},
+									"reservation_affinity": {
+										Type:         schema.TypeList,
+										Optional:     true,
+										AtLeastOneOf: gceClusterConfigKeys,
+										Computed:     true,
+										MaxItems:     1,
+										Description:  `Reservation Affinity for consuming Zonal reservation.`,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"consume_reservation_tyoe": {
+													Type:         schema.TypeEnum,
+													Optional:     true,
+													Default:      false,
+													AtLeastOneOf: schieldedInstanceConfigKeys,
```

Comment:
should this be shieldedInstanceConfigKeys ?

Reply:
It should be but the variable definition itself at ln 68 has the name